articles/quickstart/webapp/aspnet-core-3/03-authorization.md

```diff
@@ -18,57 +18,25 @@ ASP.NET Core supports [Role based Authorization](https://docs.microsoft.com/en-u
 To follow the tutorial, make sure you are familiar with [Rules](/rules/current).
 :::
 
-## Create a Rule to Assign Roles
+## Create and assign roles
 
-Create a rule that assigns the following access roles to your user:
-* An admin role
-* A regular user role
+Before we can add Role Based Access Control, we will need to ensure the required roles are created and assigned to the corresponding user(s).
+Follow the guidance explained in [assign-roles-to-users](/users/assign-roles-to-users) to ensure your user gets assigned the `admin` role.
```

Comment:
I would remove the person out of the context here and be more impertive. "Before RBAC can be added, ensure the required roles are created.."

Reply:
As mentioned on Slack, I think this is a great addition but the majority of the quickstart is already written using 'you/your' so we have decided to just ensure the added/changed text is consistent and also used 'you/your'